File: eTeatar/eTeatar.Model/Ocjena.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace eTeatar.Model
{
    public class Ocjena
    {
        public int OcjenaId { get; set; }

        public int Vrijednost { get; set; }

        public string Komentar { get; set; }

        public DateTime DatumKreiranja { get; set; }

        public int PredstavaId { get; set; }

        public int KorisnikId { get; set; }
    }
}

File: eTeatar/UI/eteatar_mobile/lib/models/ocjena.dart
import 'package:json_annotation/json_annotation.dart';

part 'ocjena.g.dart';

@JsonSerializable()
class Ocjena {
  int? ocjenaId;
  int? vrijednost;
  String? komentar;
  int? predstavaId;
  int? korisnikId;
  Ocjena(
    this.ocjenaId,
    this.vrijednost,
    this.komentar
  );

  factory Ocjena.fromJson(Map<String, dynamic> json) => _$OcjenaFromJson(json);

  /// Connect the generated [_$GlumacToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$OcjenaToJson(this);
}